include/nlmacros.h

```diff
@@ -1,6 +1,6 @@
 /*
  *
- *    Copyright (c) 2012-2018 Nest Labs, Inc.
+ *    Copyright (c) 2012-2019 Nest Labs, Inc.
```

Comment:
This should add a new "Copyright (c) 2019 Google LLC" line since Nest Labs is no more as a corporate entity.